tests/integration/contracts.test.js

```diff
@@ -750,6 +750,14 @@ describe('GET contracts/staff-register', () => {
 
       expect(response.statusCode).toBe(200);
       expect(response.result.data.staffRegister.length).toEqual(contractsList.length);
+      expect(response.result.data.staffRegister[0]).toEqual(expect.objectContaining({
```

Comment:
`should return a 400 error if missing '${param}' parameter` dans la route POST.
Je crois que ce test est redondant avec `'should not create a contract if user is not from the same company'`

Reply:
idem qu'au dessus, c'est pas comme si on avait passé un user qui n'existe pas, là le soucis c'est que le champ est required et qu'on l'a pas mis